tests/to_py.rs

```diff
@@ -2,7 +2,7 @@ use std::cmp::Ordering;
 use std::mem::size_of;
 
 use ndarray::{array, s, Array2, Array3};
-use numpy::{IntoPyArray, PyArray, ToPyArray};
+use numpy::{IntoPyArray, PyArray, PyArrayMethods, PyUntypedArrayMethods, ToPyArray};
```

Comment:
Now that I see this, could you add the conversion traits to the prelude and make use of it here?